graphbolt/src/csc_sampling_graph.cc

```diff
@@ -123,11 +123,42 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::InSubgraph(
 
 c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighbors(
     const torch::Tensor& nodes) const {
-  // TODO(#5692): implement this.
+  const int64_t num_nodes = nodes.size(0);
+
+  std::vector<torch::Tensor> picked_indices_per_node(num_nodes);
+  torch::Tensor picked_num_per_node =
+      torch::zeros({num_nodes + 1}, indptr_.options());
+
+  torch::parallel_for(0, num_nodes, 32, [&](size_t b, size_t e) {
```

Comment:
making `32` (grain_size?) as a const varaible is better, I think.

Reply:
Yes, it will be in future PR